### PR TITLE
Add alert to No Requirements toggle

### LINKED ISSUE #263: Add popup about not all cities being shown by default
Captura de pantalla 2023-10-14 a la(s) 10 46 55 p m

### PR DESCRIPTION
**Pull request by tunglinn**
Resolves #263 

New render:
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,6 +72,15 @@
       </div>
     </header>
 
+    <div class="alert" id="no-requirements-alert">
+      <span class="closebtn" onclick="this.parentElement.style.display='none';"
+        >&times;</span
```
Comment (tunglinn) [inline index.html:77]: `&times;` is the multiplication sign for the 'x' button. I like it but for the sake of standardization, should I convert it fontAwesome?

File: src/css/_alert.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  position: absolute;
+  top: 7.4em;
+  left: 50%;
+  transform: translate(-50%, 0);
+  z-index: 1100;
+
+  width: 40%;
+  @media only screen and (max-width: 48em) {
+    width: 90%;
+  }
```
Comment (tunglinn) [inline src/css/_alert.scss:13]: On small screens (e.g. mobile), the banner covers most of the horizontal.

File: index.html (hunk 1 above)
Comment (Eric-Arellano) [reply index.html:77]: I think this is okay.

File: index.html [hunk 3]
```diff
@@ -72,6 +72,15 @@
       </div>
     </header>
 
+    <div class="alert" id="no-requirements-alert">
+      <span class="closebtn" onclick="this.parentElement.style.display='none';"
```
Comment (Eric-Arellano) [inline index.html:76]: This class should be made less generic. Like `alert-close-button`.

File: src/js/filter.ts [hunk 4]
```diff
@@ -110,6 +110,19 @@ const setUpAllFilters = (
   document
     .querySelectorAll("input[type=checkbox]")
     .forEach((option: HTMLInputElement) => {
+      if (option.id == "no-requirements-toggle") {
```
Comment (Eric-Arellano) [inline src/js/filter.ts:113]: I think this code would be better in a file `alert.ts` and specifically use a query selector for `no-requirements-toggle`. It's not a big deal that we will select the element twice. 

`filter.ts` is already complex enough, and it's useful to not make it more complex.

**Review by Eric-Arellano — COMMENTED** [review]
Thank you!

File: src/js/setUpSite.ts [hunk 5]
```diff
@@ -98,6 +99,7 @@ const setUpSite = async (): Promise<void> => {
   setUpSearch(markerGroup, citiesToMarkers, data, searchElement, sliders);
   setUpAllFilters(markerGroup, citiesToMarkers, data, searchElement, sliders);
   setUpFilterPopup(markerGroup, citiesToMarkers, data, searchElement, sliders);
+  setUpAlerts();
```
Comment (Eric-Arellano) [inline src/js/setUpSite.ts:102]: Nit: I've been putting these up higher, like line 88, since they don't need any arguments to work.

**Pull request merged**